test/integration/read-write-concern/readconcern.test.ts

```diff
@@ -261,65 +237,47 @@ describe.only('ReadConcern', function () {
         { maxPoolSize: 1, readConcern: { level: 'majority' }, monitorCommands: true }
       );
 
-      client
-        .connect()
-        .then(() => {
-          // Get a collection
-          const collection = client
-            .db(configuration.db)
-            .collection('readConcernCollectionAggregate1');
+      await client.connect();
+      // Get a collection
+      const collection = client.db(configuration.db).collection('readConcernCollectionAggregate1');
 
-          // Listen to apm events
-          client.on('commandStarted', filterForCommands('aggregate', started));
-          client.on('commandSucceeded', filterForCommands('aggregate', succeeded));
+      // Listen to apm events
+      client.on('commandStarted', filterForCommands('aggregate', started));
+      client.on('commandSucceeded', filterForCommands('aggregate', succeeded));
 
-          // Execute find
-          return collection
-            .aggregate([{ $match: {} }, { $out: 'readConcernCollectionAggregate1Output' }])
-            .toArray()
-            .then(() => {
-              validateTestResults(started, succeeded, 'aggregate', 'majority');
-
-              // Execute find
-              return collection
-                .aggregate([{ $match: {} }], { out: 'readConcernCollectionAggregate2Output' })
-                .toArray()
-                .then(() => {
-                  validateTestResults(started, succeeded, 'aggregate', 'majority');
-                });
-            });
-        })
-        .then(
```

Comment:
this entire block with `client.close` can be dropped because we already do `client.close` in `afterEach`